### ISSUE TITLE
Why installing postgresql 11 if it is not supported

### ISSUE THREAD

**Issue by mishaker** [timescale/timescaledb#893]
## Summary 
`brew install timescaledb`installs postrges 11 before installing timescaledb. but then it moans because postgresql 11 is not supported. 

OS: Mac 10.12.6

```➜  Projects brew install timescaledb
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 3 taps (homebrew/cask, homebrew/cask-versions and homebrew/core).
==> Updated Formulae
libtins                                                                                   media-info

==> Installing timescaledb from timescale/tap
==> Installing dependencies for timescale/tap/timescaledb: postgresql
==> Installing timescale/tap/timescaledb dependency: postgresql
==> Downloading https://homebrew.bintray.com/bottles/postgresql-11.1.sierra.bottle.tar.gz
Already downloaded: /Users/mishaker/Library/Caches/Homebrew/downloads/3be1486d9ccbc23b4d36bf286283b2c693eb46f6388cd24d0f071f321fec8085--postgresql-11.1.sierra.bottle.tar.gz
==> Pouring postgresql-11.1.sierra.bottle.tar.gz
==> Caveats
To migrate existing data from a previous major version of PostgreSQL run:
  brew postgresql-upgrade-database

To restart postgresql after an upgrade:
  brew services restart postgresql
Or, if you don't want/need a background service you can just run:
  pg_ctl -D /usr/local/var/postgres start
==> Summary
🍺  /usr/local/Cellar/postgresql/11.1: 3,548 files, 40.3MB
==> Installing timescale/tap/timescaledb
==> Downloading https://timescalereleases.blob.core.windows.net/homebrew/timescaledb-1.0.1.tar.gz
Already downloaded: /Users/mishaker/Library/Caches/Homebrew/downloads/22d523494e39c693c04c554bae69956a0e21eef96bde3a376a876756fd23d1eb--timescaledb-1.0.1.tar.gz
==> ./bootstrap -DPROJECT_INSTALL_METHOD="brew"
Last 15 lines from /Users/mishaker/Library/Logs/Homebrew/timescaledb/01.bootstrap:
-- TimescaleDB version 1.0.1. Can be updated from version 1.0.0
-- Build type is Release
-- Install method is 'brew'
-- Performing Test CC_SUPPORTS_VISIBILITY_HIDDEN
-- Performing Test CC_SUPPORTS_VISIBILITY_HIDDEN - Success
-- Using compiler AppleClang
-- Found Git: /usr/local/Homebrew/Library/Homebrew/shims/mac/super/git (found version "2.14.3 (Apple Git-98)")
-- Using pg_config /usr/local/bin/pg_config
-- Compiling against PostgreSQL version 11.1
CMake Error at CMakeLists.txt:173 (message):
  TimescaleDB only supports PostgreSQL 9.6 or 10


-- Configuring incomplete, errors occurred!
See also "/tmp/timescaledb-20181211-53890-na8ekl/timescaledb/build/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/timescale/homebrew-tap/issues```

**Comment by RobAtticus** [comment]
Hi @mishaker , we are hoping to release a version with beta PG11 support this week, which should address this issue.

**Comment by mishaker** [comment]
Ok, I will wait till the release to test it :) thanks for your response :)

**Comment by dianasaur323** [comment]
Hi @mishaker - we just released 1.1, so I would suggest you upgrading to that and testing this out with our beta PG 11 support. Closing this out, but please feel free to open another issue if you hit any other issues.
